Add mapset banner mask support to song select UI

Janko5 opened the pull request:
Introduces a new mapset banner mask texture to the song select UI, allowing for custom masking of mapset banners. Updates DrawableBanner to use the mask if available, and adds support for loading the mask in skinning. Submodule updates for Quaver.Resources and Wobble are also included.

<img width="671" height="193" alt="Quaver_cfvNg7CW5m" src="https://github.com/user-attachments/assets/e31876d8-6f15-4574-ab64-929239c4cdcf" />

Janko5 committed:
Add mapset banner mask support to song select UI

Introduces a new mapset banner mask texture to the song select UI, allowing for custom masking of mapset banners. Updates DrawableBanner to use the mask if available, and adds support for loading the mask in skinning. Submodule updates for Quaver.Resources and Wobble are also included.

AiAe commented:
@codex review

Janko5 committed:
Merge upstream/develop into mapset-banner-mask

Warp9000 reviewed (approved):
code good but default mask should probably behave like current default skin

Janko5 committed:
Update Quaver.Resources

Janko5 committed:
Trigger build after Resources update

Janko5 committed:
Resolve Quaver.Resources pointer

Janko5 committed:
Merge upstream/develop into mapset-banner-mask

Janko5 committed:
Merge remote-tracking branch 'upstream/develop' into mapset-banner-mask

Janko5 committed:
Update submodule pointers

Advance submodule refs for Quaver.Server.Client and Wobble to pick up upstream changes. Quaver.Server.Client: fe65222ed -> 8031a47d5. Wobble: d4c16c97f -> de608c7d7.

Janko5 committed:
Add Quaver.Shared.Utils using in Program.cs

Add a using directive for Quaver.Shared.Utils in Program.cs to make utility types and methods from that namespace available in the file. This prepares the codebase for references to utilities defined in Quaver.Shared.Utils.

Janko5 committed:
Update Program.cs

Janko5 committed:
Trigger rebuild

A6sT commented:
Will be merged in #4644
